Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Júlio Hoffimann <julio.hoffimann@gmail.com> and contributors"]
 version = "1.1.0"
 
 [deps]
+CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
```

Comment:
I think we also need an entry in the [compat] section with the version of CategoricalArrays.jl introduced here. It should be the latest stable release of the package.